python/cugraph/tests/test_multigraph.py

```diff
@@ -0,0 +1,96 @@
+import cugraph
+import networkx as nx
+from cugraph.tests import utils
+import pytest
+import gc
+import numpy as np
+
+
+@pytest.mark.parametrize("graph_file", utils.DATASETS)
+def test_multigraph(graph_file):
+    gc.collect()
```

Comment:
I've started including a separate setup function that has this in it so it's not repeated in every test.

Reply:
updated